src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/Aes.cs

```diff
@@ -12,34 +10,20 @@ public abstract class Aes : SymmetricAlgorithm
     {
         protected Aes()
         {
-            this.BlockSize = 128;
-            this.KeySize = 256;
-            this.Mode = CipherMode.CBC;
-        }
-
-        public override KeySizes[] LegalBlockSizes
-        {
-            get
-            {
-                return s_legalBlockSizes.CloneKeySizesArray();
-            }
-        }
+            LegalBlockSizesValue = s_legalBlockSizes;
+            LegalKeySizesValue = s_legalKeySizes;
```

Comment:
LegalBlockSizesValue and LegalKeySizesValue are protected fields (not properties), so there is an issue here